__tests__/app.test.js

```diff
@@ -474,3 +474,136 @@ describe("10 POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe('11 - GET api/articles - with queries', () => {
+  test('200 - Returns articles sorted by sort_by column', () => {
+    return request(app)
+    .get("/api/articles?sort_by=created_at")
+    .expect(200)
+    .then(({ body: {articles} }) => {
+     expect(articles).toBeInstanceOf(Array);
+      expect(articles.length).toBe(12);
+      articles.forEach((article) => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(Number),
+          })
+        );
+      });
+      expect(articles).toBeSortedBy("created_at", { descending: true });
+    });
+});
+    
+test('200 - Returns articles sorted by sort_by column - title', () => {
+  return request(app)
+  .get("/api/articles?sort_by=title")
+  .expect(200)
+  .then(({ body }) => {
+    const { articles } = body;
+    expect(articles).toBeInstanceOf(Array);
+    expect(articles.length).toBe(12);
+    articles.forEach((article) => {
+      expect(article).toEqual(
+        expect.objectContaining({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: expect.any(Number),
+        })
+      );
+    });
+    expect(articles).toBeSortedBy("title", { descending: true });
+  });
+});
+
+test('200 - Returns articles sorted by sort_by column - votes, ASC', () => {
+  return request(app)
+  .get("/api/articles?sort_by=votes&order=asc")
+  .expect(200)
+  .then(({ body }) => {
```

Comment:
With TDD you should aim to test `order` by itself before combining it with other queries

If this test started failing (for whatever reason) you're not going to have the clearest idea of what the problem could be